lib/studies/reply/adaptive_nav.dart

```diff
@@ -241,12 +241,23 @@ class _DesktopNavState extends State<_DesktopNav>
               );
             },
           ),
-          const VerticalDivider(thickness: 1, width: 1),
+          const VerticalDivider(thickness: 1, width: 2),
```

Comment:
Whats the reason for this change?

Reply:
![white_lines](https://user-images.githubusercontent.com/948037/107474563-6528c780-6b27-11eb-97e1-2194733953ab.png)

I increased the width to remedy some artifacts (white lines shown above). From my observation, they seem to be a result of the navigation rail body not expanding to its max-width like it previously did before this change.

Reply:
These lines still show up if I don't have my browser window maximized. I'll look into it some more.